DigitalLearningSolutions.Web/Controllers/Register/RegisterInternalAdminController.cs

```diff
@@ -101,10 +101,11 @@ public async Task<IActionResult> Index(InternalAdminInformationViewModel model)
 
             if (delegateAccount == null)
             {
+                // We can have the centreSpecificEmail = null here because we already set it for the admin account
                 registrationService.CreateDelegateAccountForExistingUser(
                     new InternalDelegateRegistrationModel(
                         model.Centre!.Value,
-                        model.CentreSpecificEmail,
+                        null,

```

Comment:
I think the root of the problem is within `CreateDelegateAccountForExistingUser`. It shouldn't assume that I don't already have an admin account with a centre email set at the centre. Can we fix it there instead?

Reply:
Could you change this back so it's just setting the centre-specific email again? Then it'll be fixed when [HEEDLS-1002](https://github.com/TechnologyEnhancedLearning/DLSV2/pull/1286) is merged.